Makefile

```diff
@@ -370,6 +375,26 @@ verify-boilerplate: ## Verify boilerplate text exists in each file
 verify-container-images: ## Verify container images
 	TRACE=$(TRACE) ./hack/verify-container-images.sh
 
+.PHONY: verify-govulncheck
+verify-govulncheck: $(GOVULNCHECK) ## Verify code for vulnerabilities
+	$(GOVULNCHECK) ./...
+
+.PHONY: verify-security
+verify-security: ## Verify code and images for vulnerabilities
+	$(MAKE) verify-container-images && R1=$$? || R1=$$?; \
+	$(MAKE) verify-govulncheck && R2=$$? || R2=$$?; \
+	if [ "$$R1" -ne "0" ] || [ "$$R2" -ne "0" ]; then \
+	  echo "Check for vulnerabilities failed! There are vulnerabilities to be fixed"; \
+		exit 1; \
+	fi
+
+.PHONY: verify-flavors
+verify-flavors: $(FLAVOR_DIR) generate-flavors ## Verify generated flavors
```

Comment:
Looks like this is not part of the original cherry-pick.

@chrischdi Maybe for simplicity. Can you open PRs for all branches (including 1.8) to resync .github/workflows + the Makefile changes? (including the changes to the generate-flavors tool)

Reply:
Would that sync include the failed cherry-picks from https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2167 ? Don't want to hit those twice

Reply:
Yup. Talked to Christian. He'll do a series of cherry-picks later today to bring GitHub actions, Makefile and related scripts in sync across 1.8-1.5

Reply:
SGTM - I'd just started looking at the failing cherry-picks, luckily I saw this too :smile:

Reply:
Yup sorry missed writing this somewhere while debugging our CI env :)